File: CppPractice/Project1/Project1/Leetcode35.cpp
#include <string>

#include <vector>

#include <iostream>

#include <stdlib.h>

#include <time.h>

#include <sstream>

#include <algorithm>

#include <map>

#include <bitset>

using namespace std;


class Solution {
public:
    int searchInsert(vector<int>& nums, int target) {
		int ans = 0;
		for (int i = 0; i < nums.size(); i++)
		{
			if (nums[i]>=target)
			{
				return i;
			}

		}
		return nums.size();

    }
};

File: Pythonleetcode/Leetcode35.py
def searchInsert( nums, target):
    if len(nums) == 1:
        if target <= nums[0]:
            return 0
        else:
            return 1

    for i in range(0, len(nums) - 1):
        if target < nums[0]:
            return 0
            break
        if nums[i] == target:
            return i
            break
        if nums[i] < target and nums[i + 1] >= target:
            return i + 1
            break
    return len(nums)



nums=[1]
print(nums[0])
print(searchInsert(nums,0))
